### ISSUE TITLE
Cannot find bootstrap-markdown repository

### ISSUE THREAD

**Issue by zFlxw** [ghiscoding/angular-markdown-editor#64]
### Describe the bug

When trying to install this package from npm, there is an error that the bootstrap-markdown package cannot be found. I assume that might be the case because the owner of the repository changed.

The error:
```
 ERROR  Command failed with exit code 128: git ls-remote git+ssh://git@github.com/toopay/bootstrap-markdown.git HEAD
Host key verification failed.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

### Reproduction

Run `npm i angular-markdown-editor` (or in my case `pnpm`) in a fresh project.

### Environment Info

```shell
Angular 18.2.3,
Angular-Markdown-Editor latest
```

### Validations

- [X] Follow our [Code of Conduct](https://github.com/ghiscoding/angular-markdown-editor/blob/master/.github/CODE_OF_CONDUCT.md)
- [X] Read the [HOWTO - Step by Step](https://github.com/ghiscoding/angular-markdown-editor/wiki/HOWTO---Step-by-Step).
- [X] Check that there isn't [already an issue](https://github.com/ghiscoding/angular-markdown-editor/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/ghiscoding/angular-markdown-editor/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ghiscoding** [comment]
ah yeah it looks like `toopay` username was renamed to `refactory-id` and the reason I was using the GitHub link was because some options only exist in GitHub and were not released as a new version. So the fix should be to just rename the link

**Comment by ghiscoding** [comment]
fixed in [v3.1.1](https://github.com/ghiscoding/angular-markdown-editor/releases/tag/v3.1.1)
